### PR TITLE
dev-python/nbdime: Version bump to 0.3.0

### PR DESCRIPTION
**Pull request by marbre**
@mgorny Maybe you have the time to spend a look. I would like to bump the package, but I am not fully aware how to handle the use flag issue.

The webtools use flag is used to install nbdime-web. Starting
with 0.3.0, dependencies are fetched and installed durring the
install phase. The keyword is therefore masked.

Tests still fail as discussed in https://github.com/gentoo/gentoo/pull/3786.

### CODE REVIEW

File: profiles/arch/base/package.use.mask [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Marius Brehler <marbre@linux.sungazer.de> (22 May 2017)
+# webtools use flags installs downloads and installs dependcies via npm
+>=dev-python/nbdime-0.3.0 webtools
```
Comment (mgorny) [inline profiles/arch/base/package.use.mask]: Why in arch subdir? It's not like you're going to unmask it on some arches.

File: dev-python/nbdime/nbdime-0.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+python_test() {
+	# user.email and user.name are not configured in the sandbox.
+	git config --global user.email "you@example.com" || die
+	git config --global user.name "Your Name" || die
+
+	distutils_install_for_testing
+
+	py.test -l || die
```
Comment (mgorny) [inline dev-python/nbdime/nbdime-0.3.0.ebuild]: What does -l do? How about -v too?

Comment (marbre) [inline dev-python/nbdime/nbdime-0.3.0.ebuild]: `-l, --showlocals      show locals in tracebacks (disabled by default).` This is a takeover from upstream's `travis.yml`, but yes I can add -v too.

File: profiles/arch/base/package.use.mask (hunk 1 above)
Comment (marbre) [inline profiles/arch/base/package.use.mask]: You're right, I will move this to `profiles/base/package.use.mask`.

**Comment by marbre** [comment]
@mgrony Trying to fix testing, I noticed that I missed a new dep not yet in the tree. Therefore, I need a little longer to get this PR done properly.

**Comment by mgorny** [comment]
Ok, I'll mark it WIP then. Please remove it when done (if you can alter labels). If you can't, just ping us.

**Comment by monsieurp** [comment]
@mgorny can you have another look?

**Comment by monsieurp** [comment]
Sorry, I didn't intend to approve these changes :/

**Comment by mgorny** [comment]
@marbre, ping. You still with us?

**Comment by marbre** [comment]
@mgorny Yes, I am.

However, if you have been following my other PRs, it looks like it is not desired to get my stuff merged fast. It takes weeks or even months to approve my requests, even if the commits only introduce minor modifications. Therefore, I haven't put any further effort into this PR, as it brings some bigger modifications and the jupuyter packages in general are bumped quite fast. However, I can finish this within this month if you like. I will than also try to directly bump it to the newest version.

**Comment by mgorny** [comment]
I'm afraid that's a general problem with proxy-maint right now and I'm trying to find a good solution for it. So far I've improved bot notifications when people push PR updates and 'shouted' at some people. I'm also trying to immediately address any updates on PRs I've reviewed, and avoid reviewing too many, so if you submit an update, I will try to answer with feedback in <12hrs. You can also try to ping us on IRC to schedule 'live' review session and get things done fast.

**Comment by mgorny** [comment]
Feel free to reopen when you've decided to work on it.

**Pull request closed without merging**